.github/workflows/release-ts-api-package.yml

```diff
@@ -1,13 +1,159 @@
-name: Release Ts API package
+# Reference: ci.yml
+name: Release Ts API Package
+
 on:
   workflow_dispatch:
 
 env:
   NODE_AUTH_TOKEN: ${{ secrets.RELEASE_TS_API_PACKAGE_TOKEN }}
 
 jobs:
+  parachain-build-dev:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Free up disk space
+        if: startsWith(runner.name, 'GitHub Actions')
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: true
+          swap-storage: false
+          large-packages: false
+
+      - uses: actions/checkout@v4
+
+      - name: Pull docker image optinally
+        run: |
+          docker pull litentry/litentry-parachain:latest
+
+      - name: Save docker image
+        run: |
+          docker save litentry/litentry-parachain:latest | gzip > litentry-parachain-dev.tar.gz
+
+      - name: Upload docker image
+        uses: actions/upload-artifact@v3
+        with:
+          name: litentry-parachain-dev
+          path: litentry-parachain-dev.tar.gz
+          if-no-files-found: error
+
+      - name: Fail early
+        if: failure()
+        uses: andymckay/cancel-action@0.3
+
+  tee-build:
+    runs-on: ubuntu-latest
+    needs:
+      - parachain-build-dev
+    steps:
+      - name: Free up disk space
+        if: startsWith(runner.name, 'GitHub Actions')
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: true
+          swap-storage: false
+          large-packages: false
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+
+          driver: docker
+
+      - name: Pull and tag worker and cli image optionally
+        run: |
+          docker pull litentry/litentry-worker:latest
+          docker pull litentry/litentry-cli:latest
+
+      - run: docker images --all
+
+      - name: Save docker images
+        run: docker save litentry/litentry-worker:latest litentry/litentry-cli:latest | gzip > litentry-tee.tar.gz
+
+      - name: Upload docker images
+        uses: actions/upload-artifact@v3
+        with:
+          name: litentry-tee
+          path: litentry-tee.tar.gz
+          if-no-files-found: error
+
+      - name: Fail early
+        if: failure()
+        uses: andymckay/cancel-action@0.3 
+        
+  update-metadata:
+    runs-on: ubuntu-latest
+    needs:
+      - parachain-build-dev
+      - tee-build
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Pull polkadot image
+        run: |
+          docker pull parity/polkadot
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: litentry-parachain-dev
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: litentry-tee
+
+      - name: Load docker image
```

Comment:
I'm not sure if you need all of these.
It seems you just have to pull the docker images as you don't have to build anything locally

Reply:
Yes, we don't want to perform any compilation operations in this GHA. The goal is to pull the latest existing Docker images and tags, and then proceed to publish their TypeScript API packages.